docs/03-evaluate/01-duration-terms.adoc

```diff
@@ -22,5 +22,6 @@ Intervallschätzung, Gesetz-der-großen-Zahl
 Effort, cost, estimate, observation/measurement, assumptions, economic dimensions: Investment, yield, cost, value, Return-on-Invest (ROI), Break-Even,
 Run-the-Shop (RTC) und Develop-the-Shop (DTC) cost, Capital Expense (CapEx), Operational Expense (OpEx);
 interval estimation, law of large numbers.
+Business goals, strategic initiatives.
```

Comment:
Nach `law of large numbers` den Punkt in ein Komma umwandeln und dann die `business goals, etc.` dahinter hängen, keine neue Zeile. 

Bitte auch bei den deutschen Begriffen hinzufügen.

Reply:
ja, aber das fliegt noch raus und ich will damit schon mal merge conflicts vorbeugen.  Deutsch ist nun auch drin.